File: src/test/java/org/kokotosi/scheduler/SchedulerLibraryTest.java
// SPDX-License-Identifier: Apache-2.0

package org.kokotosi.scheduler;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class SchedulerLibraryTest {
    @Test
    void bannerExposesTheCurrentBootstrapState() {
        assertEquals("scheduler:bootstrap", SchedulerLibrary.banner());
    }
}

File: src/test/kotlin/org/kokotosi/scheduler/SchedulerLibraryTest.kt
// SPDX-License-Identifier: Apache-2.0

package org.kokotosi.scheduler

import kotlin.test.Test
import kotlin.test.assertEquals

class SchedulerLibraryTest {
    @Test
    fun `banner exposes the current bootstrap state`() {
        assertEquals("scheduler:bootstrap", SchedulerLibrary.banner())
    }
}
